test/solidity/utils/TestBase.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity ^0.8.17;
 
 import { Test } from "forge-std/Test.sol";
+// solhint-disable-next-line no-unused-import
```

Comment:
IMO I suggest to keep only necessary imports. So add imports only in files where it's needed.

Keeping this rule we avoid also the necessity of checking TestBase everytime.

Reply:
wdyt? @0xDEnYO

Reply:
What do you mean with "necessity of checking Testbase" exactly?

Reply:
The reason we did it this way is that we can easily use console in all test contracts without having to add an import (and removing it afterwards when we dont use it anymore). 

I currently do not see any negative impact from this approach but defo open for other approaches if I am missing sth

Reply:
take a look at PR https://github.com/lifinance/contracts/pull/1054/files. There we already resigned from having `LibAllowList` in TestBase and changed test files to have direct imports.

Reply:
OK, understood. 
Fine for me.